<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckOutDialog</class>
 <widget class="QDialog" name="CheckOutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="checkOutButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
border: 2px </string>
   </property>
   <property name="text">
    <string>Select Room</string>
   </property>
  </widget>
  <widget class="QComboBox" name="roomComboBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
